cmd/immuadmin/command/database.go

```diff
@@ -33,12 +34,17 @@ import (
 func addDbUpdateFlags(c *cobra.Command) {
 	c.Flags().Bool("exclude-commit-time", false,
 		"do not include server-side timestamps in commit checksums, useful when reproducibility is a desired feature")
-	c.Flags().Bool("replication-enabled", false, "set database as a replica")
+	c.Flags().Bool("replication-enabled", false, "set database as a replica") // TODO: flag name should be changed to something like `replication-is-replica`
+	c.Flags().Bool("replication-sync-enabled", false, "enable synchronous replication")
+	c.Flags().Uint32("replication-sync-followers", 0, "set a minimum number of followers for ack replication before transactions can be committed")
 	c.Flags().String("replication-master-database", "", "set master database to be replicated")
 	c.Flags().String("replication-master-address", "", "set master address")
 	c.Flags().Uint32("replication-master-port", 0, "set master port")
 	c.Flags().String("replication-follower-username", "", "set username used for replication")
 	c.Flags().String("replication-follower-password", "", "set password used for replication")
+	c.Flags().Uint32("replication-prefetch-tx-buffer-size", uint32(replication.DefaultPrefetchTxBufferSize), "maximum number of prefeched transactions")
```

Comment:
wdyt of using the already existing pattern for options

1) `replication-master-*` for master settings
2) `replication-follower-*` for follower settings

Few suggestions
`replication-prefetch-tx-buffer-size` --> `replication-follower-buffer-size`
`replication-commit-concurrency` --> `replication-follower-concurrency`
`replication-allow-tx-discarding` --> `replication-follower-discard-precommit`

Reply:
This makes it simple to understand which setting is for what when configuring the primary or follower. Now that we have so many flags, this small differentiation will help IMO

Reply:
yes, it's becoming complex as the number of flags is increasing. I also though about removing the `commit` portion in `replication-commit-concurrency`, however by then I was not fully sure there would be  a single thing to set, in this case is the number of threads requesting commit, but in the future we might also concurrently fetch transactions. Probably it's not the case and then we can remove `commit` from the flag. And then we can add the `follower` prefix to make it clear as you are proposing